util/package.go

```diff
@@ -21,8 +21,9 @@ import (
 const defaultType = "integration"
 
 var CategoryTitles = map[string]string{
-	"logs":    "Logs",
-	"metrics": "Metrics",
+	"logs":     "Logs",
+	"metrics":  "Metrics",
+	"security": "Security",
```

Comment:
@ruflin Should I also add `events` here?

Reply:
No, this is really only about the categories shown in the UI.

Reply:
ok sounds good 👍